todo/models.py

```diff
@@ -80,6 +80,7 @@ class Task(models.Model):
         null=True,
         related_name="todo_created_by",
         on_delete=models.CASCADE,
+        editable=False,
```

Comment:
I don't agree with this change - Django docs: "If False, the field will not be displayed in the admin or any other ModelForm." We DO want it displayed in the admin, and in fact should be editable in the admin (superuser needs to be able to override this sort of thing). This change does not really really address the bug. And if we were to make a model change, you'd need to include the model migration in your pull request! But please remove this line.

Reply:
Good point. IIRC, `form.is_valid()` returns `False` if you leave that out, so I'll need to dig into the Django docs to figure out an alternate solution.

Reply:
I think you might have misread. This property only controls whether people can edit the field in the admin, and has nothing to do with whether data in a form conforms to requirements.

Reply:
I'll explain as best I understand; let me know if there are any errors. 🙂 

1. We want `created_by` to be changeable by the admin.
2. However, if we set `editable=False`, then `created_by` won't show up in the admin.

Here's the issue I had.

Setup:
1. Make the proposed changes to `task_edit.html` and `list_detail.py`.
2. Try to create a task.
Result: Task is not created.

I believe this is because the form validation is failing, but not sure. Going to experiment.